### ISSUE TITLE
New keybindings/commands for navigation should be included in chat a11y help dialog

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#269245]
We introduced new commands for chat navigation. https://github.com/microsoft/vscode/pull/268404 cc @jooyoungseo and @rperez030 

We should include this in the a11y help dialog https://github.com/microsoft/vscode/blob/6303d23df74c5ad8e9e83c61baac199551417eea/src/vs/workbench/contrib/chat/browser/actions/chatAccessibilityHelp.ts#L61

cc @roblourens
